packages/react-compass/src/list/list.styles.ts

```diff
@@ -103,6 +103,18 @@ export const StyledList = styled('div', {
 
         '&.pressed': {
           backgroundColor: '$cdgBlue100',
+          '.list-text-title': {
```

Comment:
Should not use !important flag like this, this might blocks the user to customize the layout

Reply:
this commit is reverted, move to another ticket to refactor